bot-double-inner-ring/src/main/java/osm/bots/innerringbot/InnerRingApplication.java

```diff
@@ -0,0 +1,27 @@
+package osm.bots.innerringbot;
```

Comment:
it's not *The* inner ring bot, the solution to every problem that could possibly occur on inner rings
we only solve a subset of one class of problems here: duplicated geometry

also, having multiple words as one package name is tricky, as in some cases it can be hard to read or cause misunderstandings
try to name your artifact group for the tool the same way you would name a package for a class
just imagine that instead of a single application, you're creating a whole set of related tools/libraries
```suggestion
package osm.bots.rings.inner.duplicates
```